components/Flipper/FlipBased/FlipBased.tsx

```diff
@@ -78,13 +84,12 @@ function FlipBased(props: Props) {
 
     return (
         <div>
-            {isLoading ? (
-                getLoadingElement()
-            ) : auctions.length > 0 ? (
+            {isLoading ? getLoadingElement() : null}
+            {!isLoading && !hasLoadingFailed && auctions.length > 0 ? (
                 <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center', alignItems: 'stretch' }}>{auctionsElement}</div>
-            ) : (
-                <p>No auctions found</p>
-            )}
+            ) : null}
+            {!isLoading && !hasLoadingFailed && auctions.length === 0 ? <p>No auctions found</p> : null}
+            {hasLoadingFailed ? <p>An error occured while loading the auctions...</p> : null}
```

Comment:
```suggestion
            {hasLoadingFailed ? <p>An error occured while loading the auctions</p> : null}
```
the ... makes it look like there will something happen/more info to come but it isn't